infrastructure/parameters/test.parameters.json

```diff
@@ -97,6 +97,14 @@
     },
     "publicDataDbExists": {
       "value": false
+    },
+    "slackAppToken": {
```

Comment:
Under `variables` like these two?
![image](https://github.com/dfe-analytical-services/explore-education-statistics/assets/77351705/fba88d66-71e2-45e2-b415-6e80d0bf0305)

`slackAppToken` is under `parameters` in the separate files. The `parameters` blocks in the main template.json only seem to define types, rather than values
![image](https://github.com/dfe-analytical-services/explore-education-statistics/assets/77351705/f8f994b1-adb1-4ea8-a400-3d3d5fcf5bb6)

Reply:
Yeah, so I suppose I was thinking a variable called "keyVaultId" which was constructed like:

`/subscriptions/<subscriptionName>/resourceGroups/<resourceGroupName>/providers/Microsoft.KeyVault/vaults/<keyVaultName>`

And then instead of looking up the slackBotToken in each parameters file, instead move it to a single assignment in the main template like:

```
"slackAppToken": {
      "reference": {
        "keyVault": {
          "id": "<keyVaultId>"
        },
        "secretName": "ees-alerts-slackapptoken"
      }
    }
```

Reply:
Except without the angle bracket notation of course, and some concats.